Resources/Prototypes/CharacterItemGroups/musicianInstrumentsGroups.yml

```diff
@@ -0,0 +1,31 @@
+- type: characterItemGroup
+  id: LoadoutMusicianInstruments
+  maxItems: 3
+  items:
+    - type: loadout
+      id: LoadoutItemSynthesizerInstrumentMusician
+    - type: loadout
+      id: LoadoutItemMicrophoneInstrumentMusician
+    - type: loadout
+      id: LoadoutItemKalimbaInstrumentMusician
+    - type: loadout
+      id: LoadoutItemTrumpetInstrumentMusician
```

Comment:
We can probably significantly expand this list to include a majority of all instruments in the game(Barring funky things like the Human Lungs, etc), but this is good. :)